civis/io/_files.py

```diff
@@ -303,6 +303,7 @@ def civis_to_file(file_id, buf, api_key=None, client=None):
             _civis_to_file(file_id, f, api_key=api_key, client=client)
     else:
         _civis_to_file(file_id, buf, api_key=api_key, client=client)
+        buf.flush()
```

Comment:
it seems like this should be in `_civis_to_file`, no?

Reply:
> it seems like this should be in `_civis_to_file`, no?

The `with open(buf, 'wb') as f:` bit ensures the flush happens when the user passes a file path. It's only when they pass a buffer that we need the flush, although it wouldn't hurt to have an extra flush.

Reply:
should we also rename `_civis_to_file` to `_civis_to_stream` or something?

Reply:
> should we also rename `_civis_to_file` to `_civis_to_stream` or something?

I think of files in the unix sense of anything you can seek and read and write rather than filesystem files, so I don't have an issue with `_civis_to_file` as a name. I wouldn't object to `_civis_to_stream`, though.